assets/js/portfolio.js

```diff
@@ -0,0 +1,220 @@
+  // determine flex postion of image and work description
+  let flexPosition = 'start';
+  let flexPositionReverse = 'end';
+  if (i % 2 !== 0) {
+    flexPosition = 'end';
+    flexPositionReverse = 'start';
+  }
+
+  // further loop within the parent loop to break and read technologies
+  const techData = technologies(false, projects[i].technologies);
+
+  projectsItemsBuild += `<div class="works-flex">
+                          <div class="works-flex-item work-image ${flexPosition}">
+                              <img src="${projects[i].imageUrlMobile}" class="img-style" alt="${projects[i].name} project screenshot"/>
+                              <img src="${projects[i].imageUrlDesktop}" class="img-style-desktop" alt="${projects[i].name} project screenshot"/>
+                          </div>
+                          <div class="wcontainer ${flexPositionReverse}">
+                              <div class="works-flex-item">
+                                  <h3 class="intro-text-works">${projects[i].name}</h3>
+                              </div>
+                              <ul class=" works-flex-item titles-container">
+                                  <li class="list-main">${projects[i].details[0]}</li>
+                                  <li class="titles-item">${projects[i].details[1]}</li>
+                                  <li class="titles-item">${projects[i].details[2]}</li>
+                              </ul>
+                              <p class="works-flex-item works-description">
+                                  ${projects[i].smallDescription}
+                              </p>
+                              <div class="works-flex-item">
+                                  <ul class="lang-container">
+                                      ${techData[0]}
+                                  </ul>
+                              </div>
+                              <div class="works-flex-item">
+                                  <div class="btn-see">
+                                      <a href="javascript:void(0)" class="btn-see-project" project-index="${i}">See project</a>
+                                  </div>
+                              </div>
+                          </div>
+                        </div>`;
+}
+
+document.querySelector('#portfolio').innerHTML = projectsItemsBuild;
+
+/** * Dynamically adding works-flex items in works section end */
+function displayProjectDescriptionPopup(value) {
+  const techData = technologies(true, projects[value].technologiesPopup);
+  const projectPopup = document.querySelector('#projectPopup');
+  const popupContentBuilder = `<div class="popup-content">
+                                   <div class="popup-header">
+                                       <div class="popup-title">
+                                           <div class="topic">
+                                               <h1>${projects[value].name}</h1>
```

Comment:
@tufoinnkuo10 Hello! How to hardcoded or put this color in the h1 tag please? I have done in HTML and CSS some project so that it can be shown and the color in not black over there